cryptoki/tests/basic.rs

```diff
@@ -1003,3 +1003,28 @@ fn aes_gcm_with_aad() -> TestResult {
     assert_eq!(expected_cipher_and_tag[..], cipher_and_tag[..]);
     Ok(())
 }
+
+#[test]
+#[serial]
+fn get_slot_event() -> TestResult {
+    // Not implemented in SoftHSMv2
+    // https://github.com/opendnssec/SoftHSMv2/issues/370
+    let (pkcs11, _slot) = init_pins();
+    let event = pkcs11.get_slot_event().unwrap();
```

Comment:
I think `?` should work here as expected (due to [`testresult`](https://crates.io/crates/testresult)):

```suggestion
    let event = pkcs11.get_slot_event()?;
```

Reply:
Done.